chart.js

```diff
@@ -5,7 +5,7 @@
  * @external {jQuery} $ H5P.jQuery
  * @external {EventDispatcher} EventDispatcher H5P.EventDispatcher
  */
-H5P.Chart = (function ($, EventDispatcher) {
+H5P.NDLAChart = (function ($, EventDispatcher) {
```

Comment:
@boyum Are you sure you want to merge this here?

Reply:
Thanks for the review, @otacke. We do not want for the `NDLA` namespace to leak into the H5P codebase. We did the mistake of not branching out before we created the PRs, because we were certain that there would be no more changes related to NDLA.

Recently, though, we were asked to add a namespace to all NDLA's forks of H5P libraries, to let them use both NDLA and H5P versions of content types such as VT360 and Course Presentation. This has affected a few of the PRs we have submitted, and we realise that these must be changed back in order for them to be merged into H5P.

Reply:
@boyum I see.